### PR TITLE
Update AMKO docs for 1.4.1

### PR DESCRIPTION
**Pull request by hemantavi**

### CODE REVIEW

File: docs/AMKO/crds/gslbhostrule.md [hunk 1]
```diff
@@ -45,5 +45,30 @@ If this field is not provided in `GSLBHostRule`, the site persistence property w
...
+1. GSLB_ALGORITHM_CONSISTENT_HASH (needs the hash mask in the `hashMask` field).
+2. GSLB_ALGORITHM_GEO (needs the fallback algorithm settings to be specified in `geoFallback` feilds)
+3. GSLB_ALGORITHM_ROUND_ROBIN (default)
+4. GSLB_ALGORITHM_TOPOLOGY
+
+For `GSLB_ALGORITHM_GEO` as the main algorithm for a GslbService's pool, the user also needs to set the `geoFallback` settings. `geoFallback.lbAlgorithm` can have either of the two values:
+1. GSLB_ALGORITHM_CONSISTENT_HASH (needs the hash mask in `geoFallback.hashMask`)
+2. GSLB_ALGORITHM_ROUND_ROBIN
+
+For more details on which algorithms fit best to the user's needs and how to configure addtional settings in Avi for them, follow [this](https://avinetworks.com/docs/20.1/gslb-architecture-terminology-object-model/#load-balancingalgorithms-for-gslb-pool-members) link.
```
Comment (sudswasavi) [inline docs/AMKO/crds/gslbhostrule.md:71]: For more details on the algorithm that best fits the user's need and it's configuration on the Avi Controller - follow....

File: docs/AMKO/crds/gslbhostrule.md [hunk 2]
```diff
@@ -45,5 +45,30 @@ If this field is not provided in `GSLBHostRule`, the site persistence property w
...
+```yaml
+  poolAlgorithmSettings:
+    lbAlgorithm:
+    hashMask:
+    geoFallback:
+      lbAlgorithm:
+      hashMask:
+```
+
+`lbAlgorithm` takes in the name of the algorithm for the GslbService pool:
```
Comment (sudswasavi) [inline docs/AMKO/crds/gslbhostrule.md:61]: `lbAlgorithm` is used to specify the name of the algorithm.

File: docs/AMKO/crds/gslbhostrule.md [hunk 3]
```diff
@@ -45,5 +45,30 @@ If this field is not provided in `GSLBHostRule`, the site persistence property w
...
+1. GSLB_ALGORITHM_CONSISTENT_HASH (needs the hash mask in the `hashMask` field).
+2. GSLB_ALGORITHM_GEO (needs the fallback algorithm settings to be specified in `geoFallback` feilds)
+3. GSLB_ALGORITHM_ROUND_ROBIN (default)
+4. GSLB_ALGORITHM_TOPOLOGY
+
+For `GSLB_ALGORITHM_GEO` as the main algorithm for a GslbService's pool, the user also needs to set the `geoFallback` settings. `geoFallback.lbAlgorithm` can have either of the two values:
```
Comment (sudswasavi) [inline docs/AMKO/crds/gslbhostrule.md:67]: if `GSLB_ALGORITHM_GEO` is set as the main algorithm, the user also needs to specify the `getFallback` settings.

File: docs/AMKO/crds/gslbhostrule.md (hunk 1 above)
Comment (hemantavi) [reply docs/AMKO/crds/gslbhostrule.md]: Changed. Thanks!

File: docs/AMKO/crds/gslbhostrule.md (hunk 2 above)
Comment (hemantavi) [reply docs/AMKO/crds/gslbhostrule.md]: Done. Thanks!

File: docs/AMKO/crds/gslbhostrule.md (hunk 3 above)
Comment (hemantavi) [reply docs/AMKO/crds/gslbhostrule.md]: Changed it. Thanks!

**Review by sudswasavi — APPROVED** [review]

**Pull request merged**